shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -453,6 +500,7 @@ public void printUsage() {
     System.out.println("       [tail <path>]");
     System.out.println("       [touch <path>]");
     System.out.println("       [mv <src> <dst>]");
+    System.out.println("       [loadFile <path>]");
```

Comment:
Shall we call it `load` rather than `loadFile`, as you may pass a dir in `<path>` ?